core/src/main/scala/chisel3/ModuleImpl.scala

```diff
@@ -224,6 +237,10 @@ private[chisel3] trait ObjectModuleImpl {
     /** Explicitly Asynchronous Reset */
     case object Asynchronous extends Type
   }
+
+  def getModulePrefixList: List[String] = {
```

Comment:
Could use ScalaDoc

Reply:
Whoops. Those got lost in a stash.
Re-added.